### ISSUE TITLE
Search by Parameter Value

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#7346]
I'm a bit confused, and maybe just misunderstanding, but I still cannot properly filter on parameter values. Both on my own server, and on the demo server, if I search for text that's in one of the parameter columns (but NOT in the part name/description), then those parts don't come up.

For example, on the demo server if you search for 'canvas' on the Parameters page, only one result comes up that has 'canvas' in the name. However there are multiple parts that have canvas as a parameter value. Clicking the sort arrow on a column properly sorts, but I can't figure out how to filter down by specific  parameter values.

There are also no params available in the filter dropdown, as eeintech showed in their screenshot above.

_Originally posted by @second-string in https://github.com/inventree/InvenTree/issues/4823#issuecomment-2132244334_

**Comment by SchrodingersGat** [comment]
The idea here is to add the ability to search by "parameter value" in the parameteric part table

**Comment by LavissaWoW** [comment]
Yes! Preferably, I'd love if we make it so that it's like any type of webshop where you get a list of filters containing all parameters of that type.

So, say you have a parameter called "Screw size", then instead of having to type "M3", "4/40" etc, you can just select it from a table that's compiled from available parameter values.

**Comment by second-string** [comment]
@LavissaWoW's comment is a definite 'nice to have' for me as well, but I absolutely am just okay with a raw text input that matches on all parameters for any parts in the filtered list.

**Comment by SchrodingersGat** [comment]
I'm going to open this one up for funding if people want to support getting this feature available sooner

**Comment by wolflu05** [comment]
> @LavissaWoW's comment is a definite 'nice to have' for me as well, but I absolutely am just okay with a raw text input that matches on all parameters for any parts in the filtered list. 

That sounds like #5576

**Comment by mylesagray** [comment]
I threw $50 in on this issue, would love to see this!

**Comment by SchrodingersGat** [comment]
@mylesagray appreciate it! Currently a couple of other features I am giving priority to but this is also high on my list.

**Comment by SchrodingersGat** [comment]
Hey @mylesagray unfortunately polar.sh have removed their issue funding feature - meaning that the funding you committed against this issue has been "removed". This issue is still in the pipeline, perhaps you would still be willing to sponsor via other methods (but no pressure)
